### PR TITLE
assorted text fixes for the Web site

### PR DESCRIPTION
**Pull request by yole**

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ The Kotlin community is growing day by day. Join in. Take part!
...
 
 ## Other mediums
 
-- [LinkedIn](https://www.linkedin.com/groups/Kotlin-Developers-7417237?gid=7417237&mostPopular=&trk=tyah&trkInfo=tarId%3A1404329340748%2Ctas%3Akotlin%2Cidx%3A2-1-6). Mostly for job related posts. 
-- [Google+](https://plus.google.com/communities/104597899765146112928)
 - [Twitter](http://twitter.com/project_kotlin)
-
+- [LinkedIn](https://www.linkedin.com/groups/Kotlin-Developers-7417237?gid=7417237&mostPopular=&trk=tyah&trkInfo=tarId%3A1404329340748%2Ctas%3Akotlin%2Cidx%3A2-1-6). Mostly for job related posts.
+- [Google+](https://plus.google.com/communities/104597899765146112928)
+- #kotlin channel on the [freenode.net](http://freenode.net) IRC network
```
Comment (hhariri) [inline community.md]: Are we good with promoting this now? I had it on here initially but then removed it.

File: _includes/pages/index/get-kotlin.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h3 class="option-title">Compiler</h3>
                 <h3 class="option-title">IntelliJ IDEA</h3>
 
                 <p class="option-description">
-                    You can use either the OSS Community Edition or IntelliJ IDEA Ultimate.
+                    Kotlin works with the free Community Edition, IntelliJ IDEA Ultimate or Android Studio.
```
Comment (hhariri) [inline _includes/pages/index/get-kotlin.html]: actually should be an 'and' looking at it now...

**Comment by hhariri** [comment]
I've looked at the general changes, but the actual code changes in the HTML for features are hard (for me at least) to actually see with all the HTML noise around. @abreslav might want to review it all before merging too.

**Comment by abreslav** [comment]
I have no objections

**Pull request merged**